Exiled.Events/EventArgs/DyingEventArgs.cs

```diff
@@ -31,7 +31,7 @@ public class DyingEventArgs : EventArgs
         public DyingEventArgs(Player target, DamageHandlerBase damageHandler)
         {
             Handler = new CustomDamageHandler(target, damageHandler);
-            ItemsToDrop = new List<Item>(target.Items.ToList());
+            ItemsToDrop = target.ResetInventory(target.Items.ToList());
```

Comment:
Have you even tried to compile it? ``ResetInventory`` is a void method, it doesn't return anything.